doc/source/tools/libraries/index.rst

```diff
@@ -50,7 +58,7 @@ Tooling libraries
       :text-align: center
       :class-title: pyansys-card-title
 
-      Tool for defining basic variables, types, metadata, and values intended to provide interoperability between all products.
+      Tool for defining basic variables, types, metadata, and values intended to provide interoperability between all products
 
     .. grid-item-card:: PyAnsys Tools Versioning
```

Comment:
You should also add this info to the repo's main README. Info should be placed in both places